docs/examples/restore-in-pvc-template/rs-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: stash.appscode.com/v1beta1
```

Comment:
`rs` is the short name for replicaset. So, we should rename this to `restore-deployment`